project/jmh.scala

```diff
@@ -6,21 +6,22 @@ import sbt.util.Logger
 
 object RenaissanceJmh {
 
-  def generateJmhWrapperBenchmarkClass(
-    name: String,
-    packageName: String,
-    outputDir: File
-  ): File = {
+  def generateJmhWrapperBenchmarkClass(info: BenchmarkInfo, outputDir: File): File = {
+    val packageName = info.benchClass.getPackage.getName
+    val name = info.benchClass.getSimpleName
     val content = s"""
      package $packageName;
 
      import java.util.concurrent.TimeUnit;
+     import static java.util.concurrent.TimeUnit.MILLISECONDS;
      import org.openjdk.jmh.annotations.*;
      import org.renaissance.RenaissanceBenchmark;
      import org.renaissance.JmhRenaissanceBenchmark;
 
      @State(Scope.Benchmark)
      @OutputTimeUnit(TimeUnit.MILLISECONDS)
+     @Warmup(iterations = ${info.repetitions}, time = 1, timeUnit = MILLISECONDS)
+     @Measurement(iterations = ${info.repetitions / 4 + 1}, time = 1, timeUnit = MILLISECONDS)
```

Comment:
How did you choose this value ?

Reply:
In our infrastructure, we take about 20-40% of the last iterations to compute the mean value, which seems to give reasonably stable measurements.
The `+ 1` is added to circumvent situations in which `repetitions < 4`.